views/default/page/layouts/widgets/add_button.php

```diff
@@ -8,6 +8,8 @@
 	'context_stack' => elgg_get_context_stack(),
 	'show_access' => elgg_extract('show_access', $vars, true),
 	'owner_guid' => elgg_extract('owner_guid', $vars, elgg_get_page_owner_guid()),
+	'new_widget_column' => elgg_extract('new_widget_column', $vars),
+	'new_widget_position' => elgg_extract('new_widget_position', $vars),
```

Comment:
shouldn't there be a default. or at least casting for column

Reply:
no need...